lib/mvi/responses/icn_with_aaid_parser.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module MVI
+  module Responses
+    # This class takes a valid ICN with an Assigning Authority ID, and
+    # an ID status, and parses it to meet Vet360's icn_with_aaid design constraints.
+    #
+    class IcnWithAaidParser
```

Comment:
Extremely minor naming question, should this be `ICNWithAAIDParser`, in keeping with the capitalization of acronyms?